helper/config.php

```diff
@@ -184,6 +184,15 @@ public function __construct() {
                   'hasMeta'            => false,
                   'addMetaAtStartOnly' => false,
                   'refresh'            => false);
+        // firsttemplatedefinitionwins
+        $this->config ['firsttemplatedefinitionwins'] =
+            array('value'              => NULL,
+                  'DWGlobalName'       => 'firsttemplatedefinitionwins',
```

Comment:
```suggestion
                  'DWGlobalName'       => null,
```
I propose to remove the name here.

Reply:
This global setting 'firsttemplatedefinitionwins' does not exist in DokuWiki's config. Therefore, to prevent confusing, I prefer let it search for it. Might be that the rest of this config setting has some other nuances, but it looks similar as the other settings, so I expect it will work fine.